update retl extract schedule (#5510)
segmentio/segment-docs: src/connections/reverse-etl/index.md

6. In the **Select record to map and send** section, select which records to send to your destination after Segment completes extracting data based on your model. You can choose from:
      * Added records
      * Updated records
      * Added or updated records
      * Deleted records
7. Select a test record to preview the fields that you can map to your destination in the **Add test record** field.
8. Select the Schedule type for the times you want the model’s data to be extracted from your warehouse. You can choose from:
    * **Interval**: Extractions perform based on a selected time cycle.
    * **Day and time**: Extractions perform at specific times on selected days of the week.
9. Select how often you want the schedule to sync in **Schedule configuration**.
    * For an **Interval** schedule type, you can choose from: 15 minutes, 30 minutes, 1 hour, 2 hours, 4 hours, 6 hours, 8 hours, 12 hours, 1 day.
        * 15 minutes is considered real-time for warehouse syncs
    * For a **Day and time** schedule type, you can choose the day(s) you’d like the schedule to sync as well as the time.

>         * You can only choose to start the **Extraction** at the top of the hour.

Reviewer: ```suggestion
        * You can only choose to start the extraction at the top of the hour.
```

>         * Scheduling multiple **Extractions** to start at the same time inside the same data warehouse can cause extraction errors

Reviewer: ```suggestion
        * Scheduling multiple extractions to start at the same time inside the same data warehouse causes extraction errors.
```

10. Define how to map the record columns from your model to your destination in the **Select Mappings** section.
    * You map the fields that come from your source, to fields that the destination expects to find. Fields on the destination side depend on the type of action selected.
    * If you're setting up a destination action, depending on the destination, some mapping fields may require data to be in the form of an object or array. See the [supported objects and arrays for mapping](#supported-object-and-arrays).

> 11. Click **Create Mapping**.

Reviewer: ```suggestion
11. *(Optional)* Send a test record to verify the mappings correctly send to your destination.
12. Click **Create Mapping**.
```

> 12. Select the destination you’d like to enable on the **My Destinations** page under **Reverse ETL > Destinations**.

Reviewer: ```suggestion
13. Select the destination you’d like to enable on the **My Destinations** page under **Connections > Destinations> Reverse ETL**.
```

> 13. Turn the toggle on for the **Mapping Status**. Events that match the trigger condition in the mapping will be sent to the destination.

Reviewer: ```suggestion
14. Turn the toggle on for the **Mapping Status**. Events that match the trigger condition in the mapping will be sent to the destination.
```
